src/app/shared/components/label-form/component.ts

```diff
@@ -63,13 +65,18 @@ export class LabelFormComponent implements OnChanges, OnInit, OnDestroy, Control
   @Input() title = 'Labels';
   @Input() keyName = 'Key';
   @Input() valueName = 'Value';
-  @Input() labels: object;
+  @Input() labels: Record<string, string>;
+  @Input() disabledLabel: [string, string];
```

Comment:
Its not very obvious that first index is `key` and second is `value`. I think we should create a custom interface for this and use it everywhere. It will also replace the `KeyValue` type we're using from `@angular/common`. WDYT?

Reply:
@Talha-Jamil-TJ I see that this is still the same